src/Database/Log/LoggedQuery.php

```diff
@@ -73,8 +81,13 @@ protected function interpolate(): string
             if ($p === null) {
                 return 'NULL';
             }
+
             if (is_bool($p)) {
-                return $p ? '1' : '0';
+                if ($this->driver instanceof Sqlserver) {
+                    return $p ? '1' : '0';
```

Comment:
Seems we need to update the GA workflow to push coverage report for windows/sqlserver build too.